share/templates/01-General/00-Blank.mscx

```diff
@@ -5,8 +5,6 @@
     <currentLayer>0</currentLayer>
     <Division>480</Division>
     <Style>
-      <enableVerticalSpread>1</enableVerticalSpread>
```

Comment:
Why is this removed. This disables the vertical staves adjustment while is is intended to be the default

Reply:
If is is the default, it doesn need to be in the style settings, won't harm, but serves no purpose either

Reply:
But indeed ist is not, it does default to false.:
https://github.com/musescore/MuseScore/blob/2ba915f6bb080a80d8fabf97a374a811171c9e17/libmscore/style.cpp#L84
To me that seems a bug, doesn't it?

Reply:
It should be on foe every new score. not just for those created from templates, doesn't it?

Reply:
It is now...

Reply:
Hmm, no, bad idea. I'll revert that part of the change

Reply:
Indeed, bad idea :wink: I made this value default to make sure pre-3.6 score will not get vertical staves adjustment enabled unintentionally. That's why the templates were changed.

Reply:
Thanks for brining this up ;-) Saved me a brown bag I guess

Reply:
At least one :wink:

Reply:
Really, though, couldn't it have been the default for 3.6?  We explicitly keep a separate set of pre-3.6 defaults we apply explicitly upon load for older scores, that mechanism should have worked for ensuring compatibility.

Of course, I think I know the answer here: that mechanism probably didn't exist, or didn't work well when you started working on the spread algorithm.  Sopbox alert: this to me is probably the biggest mistake we made in 3.6 development.  We should have decided on and implemented our compatibility strategy first, before embarking on any further changes.  We would have saved us endless headaches as it would have been fully tested and debugged long before release, and we'd also have a more consistent story regarding these defaults.  See also the chord symbol playback defaults - it was as we approached 3.5 knowing full well 3.6 was also going to be a thing that I started sounding the alarm on this.  Oh well...

Reply:
When starting the work on vertical staves adjustment this mechanism didn't exist, wasn't even discussed. There was one compatibility issue, the chord symbol playback which gave a lot of discussion. 
Based on experiences in my previous life, the easiest way to keep compatibility was to set the default to false and update the templates. At least at the moment I was working on it. Which of course doesn't mean the new pre-3.6 defaults can't be used for vertical staves adjustment. In fact, I think it it pretty straightforward, it is only one parameter. There are no "shared" parameters.

With hindsight, we made some mistakes when starting 3.6 (this compatibility and not using the new code style for 3.6 too, causing quite some migration problems to master) but hopefully we learned from it :smiley:

Reply:
Don't get me started on that code style thing...

Reply:
Question is whether to change `enableVerticalSpread` to `true` (and remove it from the templates) for master?